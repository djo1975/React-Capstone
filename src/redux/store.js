import countrySlice from './countries/countrySlice';

const { configureStore } = require('@reduxjs/toolkit');

const store = configureStore({
  reducer: {
    countries: countrySlice,
  },
});

export default store;
