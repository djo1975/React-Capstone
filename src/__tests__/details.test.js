import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Details from '../components/details';

describe('Details', () => {
  it('should render the component', () => {
    const detail = renderer.create(
      <Provider store={store}>
        <Details />
      </Provider>,
    ).toJSON();
    expect(detail).toMatchSnapshot();
  });
});
