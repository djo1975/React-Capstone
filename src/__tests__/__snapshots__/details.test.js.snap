// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details should render the component 1`] = `null`;
